.github/workflows/coverage.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       - name: Generate code coverage
         run: |
           cargo llvm-cov nextest --workspace --lib --bins --no-default-features       \
-          --features "default chain telemetry v5 wasm-contracts wallet-integration"   \
+          --features "default chain telemetry contract wallet-integration"   \
```

Comment:
No need to specify `--features` at all 😄

Reply:
I see, you are right actually. Removed https://github.com/r0gue-io/pop-cli/pull/645/commits/036cd90e1f7c6d832381fd16720839c30735dd1c